tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -315,18 +322,28 @@ def test_transactions_scale_swarm(self):
         for flink in flinks:
             flink.start()
 
-        # Prepare topics
-        # Total number of workloads would be 2 x 5 = 10
-        workloads_per_node = 4
-        # For 5 nodes, total topics is 50 * 4 * 5 = 1000
-        # for 2xlarge 80 is a maximum
-        # 100 results in
-        # org.apache.flink.streaming.runtime.tasks.mailbox.TaskMailbox$MailboxClosedException: Mailbox is in state CLOSED, but is required to be in state OPEN for put operations.
-        topics_per_workload = 25
-        # 1 mil events total for single node, per topic
-        # Total number of events will be
-        # (1M / workloads) * topics_per_workload = 250k * 50 = 10M
-        target_total_events = 1 * 1024 * 1024
+        if not self.redpanda.dedicated_nodes:
+            # Prepare topics for docker env
+            # Total number of workloads would be 25
+            workloads_per_node = 1
+            topics_per_workload = 25
+            # 500k events total for single node, per topic
+            # Total number of events will be
+            # (500k / workloads) * topics_per_workload = 500k * 25 = ~12M
+            target_total_events = 500 * 1024
+        else:
+            # Prepare topics for EC2
+            # Total number of workloads would be 2 x 5 = 10
+            workloads_per_node = 4
+            # For 5 nodes, total topics is 50 * 4 * 5 = 1000
+            # for 2xlarge 80 is a maximum
+            # 100 results in
+            # org.apache.flink.streaming.runtime.tasks.mailbox.TaskMailbox$MailboxClosedException: Mailbox is in state CLOSED, but is required to be in state OPEN for put operations.
+            topics_per_workload = 25
+            # 1 mil events total for single node, per topic
+            # Total number of events will be
+            # (1M / workloads) * topics_per_workload = 250k * 50 = 10M
+            target_total_events = 1 * 1024 * 1024
```

Comment:
a bit confused here, 1M is across all workloads, so 

per_workload = 1M/20 = 250k, is this also per topic?

topics_per_workload = 25? whats 50 in the math above?

perhaps good to revisit this comment.

Reply:
Same here, will update